hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/metrics/RpcMetrics.java

```diff
@@ -205,7 +207,6 @@ public static TimeUnit getMetricsTimeUnit(Configuration conf) {
   // abstract class if we decide to do custom instrumentation classes a la
   // JobTrackerInstrumentation. The methods with //@Override comment are
   // candidates for abstract methods in a abstract instrumentation class.
-
```

Comment:
Could you please remove this whitespace change. I think it is causing the second checkstyle warning.